src/cleed++/Util/teebuf.hh

```
@@ -0,0 +1,56 @@
+{
+public:
+  typedef typename traits::int_type int_type;
+  
+  basic_teebuf(std::basic_streambuf<char_type, traits> * sb1,
+         std::basic_streambuf<char_type, traits> * sb2)
+    : sb1(sb1), sb2(sb2)
+  {
+  }
+  
+private:  
+  virtual int sync()
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)